rover_msgs/Obstacle.lcm

```diff
@@ -1,7 +1,6 @@
 package rover_msgs;
 
 struct Obstacle {
-	double bearing; // from straight ahead
-	double rightBearing;
-	double distance; // from straight ahead
-}
+	double bottom_left_coordinate_meters[3];
```

Comment:
this will probably break perceptions obstacle code before the new one gets PRed in, @niwhsa9 if obstacle id disabled does the perception code not try to do anything at all (like even subscribe/publish to the obstacle channel, initialize an obstacle object, etc) with this LCM?